packages/@guardian/source-react-components-development-kitchen/src/star-rating/StarRating.tsx

```diff
@@ -70,11 +70,7 @@ export const StarRating = ({
 	...props
 }: StarRatingProps): EmotionJSX.Element => {
 	return (
-		<figure
-			role="complementary"
```

Comment:
I hadn't actually considered that possibility as such. I was partly following the MDN docs which say:

For [`figure`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure):

> Permitted ARIA Roles: With no figcaption descendant: any, otherwise no permitted roles.

And for [the `complementary` role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/complementary_role):

> The complementary landmark role is used to designate a supporting section that relates to the main content, yet can stand alone when separated. These sections are frequently presented as sidebars or call-out boxes. If possible, use the HTML <aside> element instead.

So it didn't seem like the right attribute to add here for those reasons. But I hadn't considered fallback support for `<figure>` -- could you say more about what the browser support challenges for figure would be?

Reply:
hm - good question - interested in the outcome of this discussion as it could avoid confusion as to what the version bump would be if we don't add in this change. Proposing it as a separate PR might be a good option so we can get the primary PR goal of the vertical space change fix in?

Thinking about whether it should be patch/minor/major @bryophyta , I also reached the same point of uncertainty. The way I'm seeing it so far is that this constitutes a change to the internal component implementation and not so much the public facing API - it makes me think that it probably shouldn't be a major bump.

Worth also bearing in mind that consumers of the component kitchen expect components exported from here to be a "work in progress" and somewhat unstable. So we accept a degree of flexibility around changes compared to the components published as part of the core react-components library. There is a purposeful emphasis on allowing fast iterations without having to worry about a very extensive code review process :)

Reply:
I think following the MDN docs is wise and would agree that is is closer to path/minor rather than major, especially as it's part of the kitchen, as mentioned by @ob6160.

Reply:
thanks very much @ob6160 and @mxdvl, that sounds good to me!

given that the `role` seems potentially to be incorrect, I'm inclined to think that removing it shouldn't constitute a major change as well, and the point about expectation around dev-kitchen components makes sense to me, so I'll mark this as resolved.

if there are any other issues @ob6160 then please do let me know, but otherwise I think it's ready for review, from my perspective?